crates/subgraph/src/apy.rs

```diff
@@ -0,0 +1,832 @@
+use crate::{
+    types::common::{Erc20, Order, Trade},
+    vol::{get_vaults_vol, VaultVolume},
+    OrderbookSubgraphClientError,
+};
+use alloy::primitives::{
+    utils::{format_units, parse_units, ParseUnits, Unit, UnitsError},
+    I256, U256,
+};
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, str::FromStr};
+use typeshare::typeshare;
+
+pub const ONE: &str = "1000000000000000000";
+pub const DAY: u64 = 60 * 60 * 24;
+pub const YEAR: u64 = DAY * 365;
+pub const PREFERED_DENOMINATIONS: [&str; 11] = [
```

Comment:
don't hardcode denominations

Reply:
these are like as preferred denominations, like if an order has some io tokens, and one of them is one of these prefered ones, we would like to use that as denomination, as I thought if possible we would like to use these major tokens as denomination if possible, isnt that desireable?
its easy to remove, im just asking/suggesting

Reply:
as in, let's say we have calculated the apy in usdt, quick and ioen, for an order, and netvol is + for usdt and quick and - for ioen, we set aside ioen, and between usdt and quick which both are +, isnt it better to choose usdt over quick even if quick is more + than usdt?